app/models/customer.py

```diff
@@ -1,4 +1,23 @@
+from flask import current_app
 from app import db
+from datetime import date
+from flask import Blueprint, jsonify, request
+
 
 class Customer(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    customer_registration = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, default=date.today())
+    # videos = db.relationship("Video", secondary="rental", backref="customers")
+
+    def customer_information(self):
```

Comment:
wait you already had an instance method!! There were other places you could've used it 👀